<p>
Jump statements (<code>return</code>, <code>exit</code>, <code>continue</code>, and <code>raise</code>) move control flow out of the current code block.
Typically, any statements in a block that come after a jump are simply wasted keystrokes lying in wait to confuse the unwary.
</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
begin
  raise my_error;
  log('finished'); -- this code will never be executed
end;
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
begin
  raise my_error;
end;
</pre>
