<p>
A chain of <code>if</code>/<code>elsif</code> statements is evaluated from top to bottom.
At most, only one branch will be executed: the first one with a condition that evaluates to <code>true</code>.
</p>

<p>
Therefore, duplicating a condition automatically leads to dead code. Usually, this is due to a copy/paste error.
At best, it's simply dead code and at worst, it's a bug that is likely to induce further bugs as the code is maintained, 
and obviously it could lead to unexpected behavior.
</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
IF (param = 1) THEN
  open();
ELSIF (param = 2) THEN
  close();
ELSIF (param = 1) THEN // Incorrect
  move();
END IF;
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
IF (param = 1) THEN
  open();
ELSIF (param = 2) THEN
  close();
ELSIF (param = 3) THEN
  move();
END IF;
</pre>
