<p>
The forms "<>", "!=", "~=" and "^=" are equivalent, but only the "<>" form is defined in ANSI SQL.
</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
if a != b then
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
if a <> b then
</pre>
