<p>
Return of boolean literal statements wrapped into IF-THEN-ELSE ones should be simplified.
</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
IF expression THEN
  RETURN TRUE;
ELSE
  RETURN FALSE;
END IF;
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
RETURN expression;
</pre>
