<p>
When a SELECT INTO statement causes the predefined exception TOO_MANY_ROWS the values of the variables in the INTO clause are undefined.
</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
begin
  select empno
    into var
    from emp;
exception
  when too_many_rows then
    null;
end;
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
begin
  select empno
    into var
    from emp;
exception
  when too_many_rows then
    var := null;
end;
</pre>
