<p>Useless parentheses can sometimes be misleading and so should be removed.</p>

<h2>Noncompliant Code Example</h2>

<pre data-diff-id="1" data-diff-type="noncompliant">
if ((a = b)) then ...
</pre>

<h2>Compliant Solution</h2>

<pre data-diff-id="1" data-diff-type="compliant">
if (a = b) then ...
</pre>
